(See Desc) Dex Update

matteoruscitti opened the pull request:
Adds:
- Roedrunna
- Maliceros
- Breechon, Nimbichon
- Leviorca
- Squimshi, Gummelid
- Alivolt
- Fintricity
- Terrator
- Gengar Alola
- Intricat
(The latter two can be found exactly where their prevos are. Intricat evolves from Delcatty. I need a review on Gengar-Alola. There are still Pokemon missing. Trapinch-Orre line is missing, as is Vikavolt-Unova line. _In order for this update to be marked done, we must also complete tiering data and learnsets._ **Do not push this change until this is completed.**

**DEADLINE IS MAY 25, 2022**

matteoruscitti committed:
(See Desc) Dex Update

Adds:
- Roedrunna
- Maliceros
- Breechon, Nimbichon
- Leviorca
- Squimshi, Gummelid
- Alivolt
- Fintricity
- Terrator
- Gengar Alola
- Intricat
(The latter two can be found exactly where their prevos are. Intricat evolves from Delcatty. I need a review on Gengar-Alola. There are still Pokemon missing. Trapinch-Orre line is missing, as is Vikavolt-Unova line. In order for this update to be marked done, we must also complete tiering data and learnsets. Do not push this change until this is completed.

DEADLINE IS MAY 25, 2022

GamR-the-Great commented on data/pokedex.ts:
```diff
@@ -19937,4 +19967,124 @@ export const Pokedex: {[speciesid: string]: SpeciesData} = {
+		name: "Breechon",
+		types: ["Flying"],
+		baseStats: {hp: 65, atk: 70, def: 40, spa: 47, spd: 30, spe: 70},
+		abilities: {0: "Cloud Nine", 1: "Fur Coat", H: "Aerilate"},
```
Breechon and Nimbichon are part of the same line, but you missed the evo & prevo stuff completely here.

GamR-the-Great commented on data/pokedex.ts:
```diff
@@ -19937,4 +19967,124 @@ export const Pokedex: {[speciesid: string]: SpeciesData} = {
+		newmeta: "true",
+	},
+	gummelid: {
+		num: 10098,
```
Same with Squimshi & Gummelid

GamR-the-Great committed:
Additonal changes to pokedex.ts

jmf61 committed:
tiering for roedrunna through terrator

jmf61 commented:
Added the tiering for Roedrunna through Terrator, still need tiering for Gengar-Alola and all mons listed below

jmf61 committed:
Learnsets for roedrunna through terrator

jmf61 commented:
Added learnsets for Roedrunna through Terrator, still need learnsets for Gengar-Alola and all mons listed below

matteoruscitti commented:
fintricity has "hydropum" rather than hydropump. Did the tool generate the miss spell or were these typed manually?

matteoruscitti committed:
Update formats-data.ts

jmf61 commented:
> fintricity has "hydropum" rather than hydropump. Did the tool generate the miss spell or were these typed manually?

the submission had it misspelled, so autoformatting meant the code had same misspell

jmf61 committed:
Merge branch 'master' into matteoruscitti-may25-2022-update

jmf61 committed:
fix hydropump misspelling

matteoruscitti committed:
Adding Regional Variants + Intricat Tiering

Adds:
- Intricat
- Gengar-Alola
- Grubbin, Charjabug, Vikavolt Unova
- Trapinch, Vibrava, Flygon Orre
- Ferroseed, Ferrothorn Kliiro

Cheers.
